packages/node_modules/@webex/plugin-meetings/src/locus-info/controlsUtils.js

```diff
@@ -0,0 +1,34 @@
+const controlsUtils = {};
```

Comment:
Not a blocker, but to match the other util files in the meetings plugin, this should be capitalized.

```suggestion
const ControlsUtils = {};
```

Reply:
hostUtils, selfUtils, mediaSharesUtils etc are all camel case

Reply:
In locus-info you are importing them like this: 
```js
import ControlsUtils from '../locus-info/controlsUtils';
import InfoUtils from '../locus-info/infoUtils';
import FullState from '../locus-info/fullState';
import SelfUtils from '../locus-info/selfUtils';
import HostUtils from '../locus-info/hostUtils';
``` 
I'd like for them to be consistent with how they are imported. No need to fix the others in this PR, though.